scripts/rtl872x.tcl

```diff
@@ -153,19 +155,20 @@ proc rtl872x_init {} {
         echo "initializing rtl872x"
         init
         reset halt
+				#write_memory 0x00082000 32 rtl872x_flash_loader 
         array2mem rtl872x_flash_loader 32 0x00082000 [array size rtl872x_flash_loader]
```

Comment:
This looks like the correct syntax. Is this not working?

Reply:
I had problems with the progress bar when I did this change. 
After 100% of flashing it loops to 120..130... 150% and so on and so forth so idk why this behavior that's why I commented this instruction hehe.